main/acle.md

```diff
@@ -2526,11 +2527,12 @@ the [[cxxabi]](#cxxabi), and it is defined as follows:
 ```
 
 If multiple features are requested then those shall be appended in lexicographic
-order and prefixed with `M`.
+order and prefixed with `M`. The mangled name shall contain a unique set of
+features (duplication of features is not allowed).
 
 For example:
 ``` c
-__attribute__((target_clones("crc32", "aes+sha1")))
+__attribute__((target_clones("crc32", "aes+sha1+aes")))
```

Comment:
I will remove this change since it implies that the attribute string will accept duplicates, which isn't the case for every toolchain.